src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #000;
+  --hover-color: #00acdc;
+  --header-background-color: #fff;
+}
+
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
```

Comment:
You've set margins to 0 for both html and body. It would be more efficient to merge these selectors since they share the same properties.